internal/common/function/function.go

```diff
@@ -119,9 +119,9 @@ func GenerateNacObjectNameWithUUID(namespace, nacName string) string {
 	return nacObjectName
 }
 
-// GetObjectByLabel retrieves a list of Kubernetes objects of a specified type based on a label within a given namespace.
-// It returns a slice of the specified object type and an error.
-func GetObjectByLabel(ctx context.Context, clientInstance client.Client, namespace string, labelKey string, labelValue string, objectList client.ObjectList) error {
+// ListLabeledObjectsInNamespace retrieves a list of Kubernetes objects in a specified namespace
+// that match a given label key-value pair.
+func ListLabeledObjectsInNamespace(ctx context.Context, clientInstance client.Client, namespace string, labelKey string, labelValue string, objectList client.ObjectList) error {
```

Comment:
NIT: `ListObjectsByLabel`

Reply:
I wanted to be super precise what this function does ;)

Reply:
Changed to `ListObjectsByLabel`